homeassistant/components/opentherm_gw/strings.json

```diff
@@ -309,6 +309,11 @@
       "outside_temperature": {
         "name": "Outside temperature"
       }
+    },
+    "switch": {
+      "central_heating_override_n": {
+        "name": "Force central heating {circuit_number} on"
```

Comment:
```suggestion
        "name": "Force central heating {circuit_number}"
```

Reply:
I can see advantages of both. I added the `on` at the end because we can only force the circuit to remain `on`.
If the switch is turned `on`, the circuit is enabled on the boiler regardless of thermostat demand.
If the switch is turned `off`, the thermostat controls the circuit so it may still be turned `on` (however, this is only visible on the boiler `sensor` entity for that circuit and won't affect the switch state as the override is still `off`).

Reply:
Oh then `on` makes sense